config/locales/diaspora/en.yml

```diff
@@ -1008,8 +1008,13 @@ en:
 
   profiles:
     edit:
-      your_public_profile: "Your public profile"
-      your_private_profile: "Your private profile"
+      basic: "Basic"
+      additional: "Additional"
```

Comment:
How about 'Your basic profile'/'Your core profile' and 'Your detailed profile'/'Your extended profile'?

Would 'My' read better than 'Your' in these strings? It can make the page feel more like something created by the user themselves, rather than someone else (the server) talking to them. There are pros and cons on both sides, I think. What do others think?

Will the basic profile fields (ID, name, avatar, tags) continue to be public, or will these fields also be optionally public/private? If they will remain public, which seems to me the sensible thing, it might be worth adding a note to this effect under the Profile tab at the top of the page. Perhaps something like:

> Every item in your profile is optional. Your basic profile will always be publicly visible; each item in your extended [or detailed] profile can be set as public or private by you.

One other thing: is 'private' the right word here? It's not being kept private; it's being given a limited scope, rather than public. Perhaps 'limited' or 'contacts only' would be more accurate.